config/config.go

```diff
@@ -44,19 +44,25 @@ type Config struct {
 	SwaggerApiEnabled bool
 
 	// XDP Root program details.
-	XDPRootProgramName     string
-	XDPRootProgramArtifact string
-	XDPRootProgramMapName  string
-	XDPRootProgramCommand  string
-	XDPRootProgramVersion  string
+	XDPRootProgramName       string
+	XDPRootProgramArtifact   string
+	XDPRootProgramMapName    string
+	XDPRootProgramCommand    string
+	XDPRootProgramVersion    string
+	XDPRootProgramObjectFile string
+	XDPRootProgramProgName   string
```

Comment:
What is the difference between "ProgramName" and "ProgramProgName"?  These names sound synonymous.

Reply:
Here eBPF Program package name was referred to as ProgramName. The ProgramProgName refers to the name of the program that is being loaded.

Reply:
Too confusing.  Please fix the names to be distinct.   Neither use the word "Package" at present.